A79-non-per-call-metrics-architecture.md

```diff
@@ -253,9 +258,10 @@ implementations can choose one of two approaches.
 In the first approach, stats plugins provide a scope filter that applications
 can configure. Based on the channel's configuration, a stats plugin can decide
 whether it is interested in the channel. At channel creation time, the channel's
-target and the authority is provided to the stats plugin to make this decision.
-Various gRPC components use this list of interested stats plugins
-(`StatsPluginGroup`) to record metrics.
+canonical target and the default authority is provided to the stats plugin to
+make this decision. (The default authority provided is derived from the target
+unless an override was specified on the channel) Various gRPC components use
```

Comment:
Nit: Need a period before the close-paren.

Reply:
hawkeye :)